resources/src/IconActionSearch.tsx

```diff
@@ -0,0 +1,24 @@
+import { Search } from '@geist-ui/react-icons'
+import { Tooltip, useToasts } from '@geist-ui/react'
+
+const IconActionSearch = (props: { side?: boolean }) => {
+  const [, setToast] = useToasts()
+
+  const placement = props.side ? 'left' : 'top'
+
+  return (
+    <Tooltip text="Search" placement={placement} enterDelay={0}>
+      <Search
+        className="cursor-pointer"
+        size={28}
+        onClick={() => {
+          setToast({
+            text: "Use your browser's search functionality (Ctrl + F) to search",
```

Comment:
I'm not sure how much work it would be, but it would be nice to detect the OS and say `Cmd + F` if macOS else `Ctrl + F` (fyi Mac keyboards use `cmd` (command) instead of `ctrl` (control)).

Reply:
I guess we can find a React hook for this problem (or create one our self). ^^

Reply:
It's really minor; don't trouble yourself.

Reply:
I'm using the library [`react-device-detect`](https://www.npmjs.com/package/react-device-detect) to detect if the device is made by Apple. In this case a shortcut is displayed using the [`Keyboard`](https://react.geist-ui.dev/en-us/components/keyboard) component of Geist UI. Else the hotkey for other systems is printed as text, because it looks better.

It would be nice if you could test this feature.

Reply:
It works beautifully 🎉

>Use your browser's search functionality `⌘ f` to search

Reply:
Great 😄